components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3467,6 +3467,7 @@ subroutine cloud_sedimentation(kts,kte,ktop,kbot,kdir,   &
 
       endif two_moment
 
+      ! JGF: Is prt_liq intended to be inout or just out? Inconsistent with rain and ice sed.
       prt_liq = prt_accum*inv_rhow*odt  !note, contribution from rain is added below
```

Comment:
@pressel , @AaronDonahue , I noticed this during this PR. The other sedimentation routines add onto their prt_XXX parameters, which cloud_sedimentation just sets it. Is this intentional?

Reply:
Good eyes. My guess is that b/c cloud comes first, prt_liq is 0 on input. But for consistency among functions, we could consider +=, which would be BFB the same if my guess is right.

Reply:
Yeah, I'm thinking we either need to += or change the param to intent(in)